app/src/main/java/com/android/solvit/seeker/ui/profile/EditSeekerPreferences.kt

```diff
@@ -0,0 +1,175 @@
+                        "🏋️ Personal Training",
+                        "✍️ Writing")
+                val remainingSuggestions = allSuggestions.filterNot { it in selectedItems }
+
+                Text(
+                    text = "Selected preferences:",
+                    fontSize = 18.sp,
+                    fontWeight = FontWeight.Bold,
+                    modifier =
+                        Modifier.align(Alignment.Start).testTag("selected_preferences_title"))
+                Spacer(modifier = Modifier.height(8.dp))
+
+                SuggestionsGrid(
+                    suggestions = selectedItems,
+                    selectedItems = selectedItems.toSet(),
+                    onSuggestionClick = { suggestion ->
+                      viewModel.deleteUserPreference(userId, suggestion)
+                    })
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                Text(
+                    text = "You might like:",
+                    fontSize = 18.sp,
+                    fontWeight = FontWeight.Bold,
+                    modifier =
+                        Modifier.align(Alignment.Start).testTag("available_preferences_title"))
+                Spacer(modifier = Modifier.height(8.dp))
+
+                SuggestionsGrid2(
+                    suggestions = remainingSuggestions,
+                    selectedItems = selectedItems.toSet(),
+                    onSuggestionClick = { suggestion ->
+                      viewModel.addUserPreference(userId, suggestion)
+                    })
+              }
+
+          // The button always at the bottom
+          Button(
+              onClick = { navigationActions.goBack() },
+              modifier =
+                  Modifier.align(Alignment.BottomCenter) // Correct alignment here
+                      .height(60.dp)
+                      .testTag("updatePreferencesButton"),
+              colors = ButtonDefaults.buttonColors(colorScheme.secondary)) {
+                Text(text = "Update Preferences")
+              }
+        }
+      }
+}
+
+@Composable
+fun SuggestionsGrid2(
+    suggestions: List<String>,
+    selectedItems: Set<String>,
+    onSuggestionClick: (String) -> Unit
+) {
+  Column(modifier = Modifier.fillMaxWidth()) {
+    val chunkedSuggestions = suggestions.chunked(2) // Divide into rows of 2 items
+    chunkedSuggestions.forEach { rowItems ->
+      Row(horizontalArrangement = Arrangement.Start, modifier = Modifier.fillMaxWidth()) {
+        rowItems.forEach { suggestion ->
+          SuggestionButton2(
+              text = suggestion,
+              isSelected = selectedItems.contains(suggestion),
+              onClick = { onSuggestionClick(suggestion) })
+        }
+      }
+    }
+  }
+}
+
+@Composable
+fun SuggestionButton2(text: String, isSelected: Boolean, onClick: () -> Unit) {
+  // Update color based on selection state
+  val backgroundColor = if (isSelected) colorScheme.secondary else Color(0xFFF1F1F1)
```

Comment:
Again think to use material theme file so that our app is more consistent and above all if we decide maybe to change the colours before the BOTA, that would make the task easier

Reply:
You can directly define the colour in file colours

Reply:
I fixed that.